### ISSUE TITLE
Can someone please archive?

### ISSUE THREAD

**Issue by wertzuz** [bountysource/core#1599]

**Comment by tuxayo** [comment]
@wertzuz No, that would prevent scammed people to communicate there. Suggestion: close the issue and change the title to have less chances of archiving happening.

**Comment by kunalsharma03052008-cell** [comment]
Don’t archive or lock the issue immediately, as that would block affected users (e.g., people who were scammed) from sharing their experiences or updates.
Instead, close the issue but keep it accessible for visibility.
Update the title to be more neutral or less likely to trigger automated archiving or moderation filters.
This keeps the discussion discoverable while reducing the risk of it being auto-archived.
